### PR TITLE
comment and chunck option on the ANCOVA part

### PR DESCRIPTION
**Pull request by ManonDenis**
I have modified the comment and add chunck option (hide the code, label the figure...) on the ANCOVA part

### CODE REVIEW

**Review by Antoine1736 — APPROVED** [review]

**Review by KlerviChoulette — APPROVED** [review]

**Comment by MarieEtienne** [comment]
It is strange that there is no mention of the verification (check). It is difficult to be sure that the workflow is correct. You should try to close and repoen you issue to be sure that the checks are running. 
If not, it is probably because you have introduced bugs in the website.yml file (in .github/workflows/website.yml)

File: .github/workflows/website.yml [hunk 1]
```diff
@@ -18,6 +18,7 @@ jobs:
       - name: Install R packages
         run: |
           Rscript -e "install.packages(c('factoextra'))"
+	  Rscript -e "install.packages(c('quarto'))"
```
Comment (MarieEtienne) [inline .github/workflows/website.yml:21]: this should not be mandatory. quarto is available by default as an external program

Comment (MarieEtienne) [reply .github/workflows/website.yml:21]: ok I see you didn't respect the space formatting

**Review by celinetanguy — APPROVED** [review]

**Review by ArthurMotte — APPROVED** [review]
Is the anova part changed from what have done Celine? Nice for ANCOVA

**Pull request merged**